src/stories/containers/Finances/components/HeaderTable/HeaderQuarterly/CellQuarterly.tsx

```diff
@@ -0,0 +1,152 @@
+import styled from '@emotion/styled';
+import { returnShortNameForMetric } from '@ses/containers/Finances/utils/utils';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+import lightTheme from '@ses/styles/theme/light';
+import React from 'react';
+import type { MetricsWithAmount } from '@ses/containers/Finances/utils/types';
+import type { WithIsLight } from '@ses/core/utils/typesHelpers';
+
+interface Props {
+  metrics: MetricsWithAmount[];
+  quarterly: string;
```

Comment:
Would be a good idea to use the `enum` I propose in my previous comment here too

Reply:
For this , quarterly , its indeed an string, its not ('Annually', 'Quarterly', etc.), instead its the string with the form of (Q1-2023,Q2-2023,Q3-2023,Q4-2023)